packages/syft/src/syft/core/node/common/client_manager/user_api.py

```diff
@@ -45,10 +59,17 @@ def create(self, **kwargs: Any) -> None:
                 )  # type: ignore
                 logger.info(response)
             else:
-                response = self.perform_api_request(
-                    syft_msg=self._create_message, content=kwargs
-                )
-                logger.info(response.resp_msg)
+                if flags.USE_NEW_SERVICE:
+                    response = self.perform_request(
```

Comment:
I guess this is the Python API, so it is not covered by the current backend tests?
Perhaps we can make sure this is tested in the next iteration of the Service refactor by adding some Integration tests with `USE_NEW_SERVICE=True`.